nbs/getting_started.ipynb

```diff
@@ -28,21 +28,21 @@
     "\n",
```

Comment:
<p>"Video walkthrough" has a period at the end. </p><p> </p><p>Do you think there is a way to avoid repeating yourself here.  For example:</p><p> </p><p>The best way to learn about nbdev is to either complete the [written tutorial] or this video tutorial ....:</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/925/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>